### DISCUSSION TITLE
Multi tenant

### DISCUSSION THREAD

**Discussion by jimmysafe** [fattureincloud/api#473] [Q&A] [answered]
Abbiamo un software con un centinaio di utenti e vorremmo abilitarli all'invio delle fatture al SDI tramite l'api di Fatture in cloud.

Esiste un modo e un piano di abbonamento specifico che ci metterebbe in condizione di non dover creare un account fatture in cloud per ogni utente che abbiamo?

**Comment by emajo** [comment, marked as answer]
Ciao @jimmysafe, no purtroppo non abbiamo questo tipo di abbonamento, tutti gli utenti dovranno creare un account Fatture in cloud.

Grazie,
Emanuele.
